server/components/auth/logout.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Created by sourabh on 27/7/17.
+ */
+
+const logout=(req,res)=>{
```

Comment:
Add documentation for function:
/**
* <Purpose of function>
* @params - <parameters>
* @return - <return object>
*/